src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/suppresswarningsholder/InputSuppressWarningsHolder1.java

```diff
@@ -0,0 +1,26 @@
+/*
+com.puppycrawl.tools.checkstyle.checks.SuppressWarningsHolder
+aliasList = (default)
+
+com.puppycrawl.tools.checkstyle.filters.SuppressWarningsFilter
+
+com.puppycrawl.tools.checkstyle.checks.coding.UnusedLocalVariableCheck
+
+*/
+
+//non-compiled with eclipse: The value for annotation attribute must be a constant expression
```

Comment:
Can we avoid this ?
Better to not use this conflict of jdk and eclipse compiler

Reply:
Done I think ci will not fail

Reply:
This comment was for annotation like `@SuppressWarnings({InputSuppressWarningsHolder9.unusedLocalVariableCheck`

Please simplify Input to do `@SuppressWarnings({"brr"})`

Reply:
No I can't do that this type of test is created to kill 
https://github.com/checkstyle/checkstyle/blob/0831953d82df93e5ebe7f178c2f2b904af8b8f3b/src/main/java/com/puppycrawl/tools/checkstyle/checks/SuppressWarningsHolder.java#L521-L526

> @SuppressWarnings({"brr"}) 

if I change input like this then the ast will change and token will be of type 
https://github.com/checkstyle/checkstyle/blob/0831953d82df93e5ebe7f178c2f2b904af8b8f3b/src/main/java/com/puppycrawl/tools/checkstyle/checks/SuppressWarningsHolder.java#L516
which is already covered by existing input

Reply:
Ok, in this case we need such comment, please restore it.
And we need to move file to u compiled file folder, example https://github.com/checkstyle/checkstyle/blob/63d82894e18ce12662521ce7d7042c4b214c84be/src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/grammar/InputSemicolonBetweenImports.java#L12

Please install Eclipse to confirm it is still issue for it.
https://github.com/search?q=repo%3Acheckstyle%2Fcheckstyle%20non-compiled%20with%20eclipse&type=code

Reply:
https://checkstyle.org/eclipse.html

Reply:
@romani I have updated pr after successfully verifying mvn in eclipse id